File: 8KYU/Sum of Multiples/solution.cs
using System;
using System.Linq;
public class Kata
{
  public static int SumMul(int n, int m)
  {
    Console.WriteLine($"{n} {m}");
    if(n <= 0 || m <= 0 || m < n || m == n)
      throw new ArgumentException();
    return Enumerable.Range(n, (m-n))
                     .Where(x => x % n == 0)
                     .Sum();
  }
}

File: 8KYU/Sum of Multiples/solution.rb
def sum_mul(n, m)
  if n <= 0 or m <= 0 or m == n or m < n
    return "INVALID"
  end
  result = 0
  
  for i in 1..m-1
    if i % n == 0
      result += i
    end
  end
  
  result
end